src/server/entities/invitationCode/api.js

```diff
@@ -0,0 +1,24 @@
+import InvitationCode from './model'
+
+const api = {
+  async getInvitationCode(code) {
+    return await InvitationCode.findOne({ code }).exec()
+  },
+  async claimedCode(code, name) {
+    return await InvitationCode.findOneAndUpdate(
+      { code: code },
+      {
+        isClaimed: true,
+        claimer: name
+      },
+      { new: true },
+      function(err, res) {
```

Comment:
This callback seems redundant to me, if there is an error it should be thrown anyway.